File: Java/2180.java
class Solution {
    public int countEven(int num) {
        int ans=0;
        for(int i=2; i<=num; i++) {
            if(isEven(i)) {
                ans++;
            }
        }
        return ans;
    }
    
    public boolean isEven(int n) {
        int sum=0;   
        while(n!=0) {
            sum += n%10;
            n = n/10;
        }
        return sum%2 == 0;
    }
}

File: Python/2180.py
class Solution:
    def countEven(self, num: int) -> int:
        count = 0
        for i in range(num+1):
            total = sum(int(digit) for digit in str(i)) 
            if total % 2 == 0 and total != 0:
                count += 1
        return count